README.md

```diff
@@ -122,7 +129,7 @@ meeting invite and Zoom links to join the meeting.
 [getting-started-guide]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/getting-started/
 [badge-travis-image]: https://travis-ci.org/Kong/kubernetes-ingress-controller.svg?branch=master
 [badge-travis-url]: https://travis-ci.org/Kong/kubernetes-ingress-controller
-[bintray-builds]: https://bintray.com/kong/kubernetes-ingress-controller/kong-ingress-controller
+[docker-images]: https://hub.docker.com/repository/docker/kong/kong
```

Comment:
That link requires login does it not? should we point to the org instead? https://hub.docker.com/u/kong

Reply:
Indeed. Didn't realize that was a private URL, and hadn't created the repo yet, so double-whammy bad link. Having no created the repo, 19ee1fd